Pseudocode/Dynamic Programming/Longest Palindromic Substring/SourceCode.tex

```diff
@@ -0,0 +1,72 @@
+% Set the Page Layout
+\documentclass[12pt]{article}
+\usepackage[inner = 2.0cm, outer = 2.0cm, top = 2.0cm, bottom = 2.0cm]{geometry}
+
+
+% Package to write pseudo-codes
+\usepackage{algorithm}
+
+% Remove the 'end' at the end of the algorithm
+\usepackage[noend]{algpseudocode}
+
+% Define Left Justified Comments
+\algnewcommand{\LeftComment}[1]{\Statex \(\triangleright\) #1}
+
+% Remove the Numbering of the Algorithm
+\usepackage{caption}
+\DeclareCaptionLabelFormat{algnonumber}{Algorithm}
+\captionsetup[algorithm]{labelformat = algnonumber}
+
+\begin{document}
+
+\begin{algorithm}
+
+  \caption{Find the longest palindromic sequence in a given string}
+  
+  \begin{algorithmic}[1]
+    \Ensure Zero Based Indexing
+    \Statex
+    
+    \Function{Longest\_Palindrome}{$str$}
```

Comment:
Use Proper function name. Moreover, the function name is printed in all caps. To overcome this, use Math Mode

Reply:
I didn't know about the effect of not using Math Mode in function name. I'll do that from now on.